api/FileSystemHandle.json

```diff
@@ -11,7 +11,7 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "111"
```

Comment:
LGTM, but it seems Gecko supports one more function on FileSystemHandle: `move()`. Should it added here? https://searchfox.org/mozilla-central/rev/3002762e41363de8ee9ca80196d55e79651bcb6b/dom/webidl/FileSystemHandle.webidl#16-22

Reply:
Yes, done in https://github.com/mdn/browser-compat-data/pull/19056/files#r1131816623